akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -322,6 +320,9 @@ object Flow {
    */
   def fromSinkAndSourceMat[I, O, M1, M2, M](sink: Graph[SinkShape[I], M1], source: Graph[SourceShape[O], M2])(combine: (M1, M2) ⇒ M): Flow[I, O, M] =
     fromGraph(GraphDSL.create(sink, source)(combine) { implicit b ⇒ (in, out) ⇒ FlowShape(in.in, out.out) })
+
+  def monitor[T]: Flow[T, T, FlowMonitor[T]] =
+    fromGraph(new MonitorFlow[T])
```

Comment:
I think it belongs in `FlowOps` which should make it available from the right places (and that's where all the other flow stages are available). This will make it available from instances of `Flow` and `Source` as those inherit FlowOps. 

I was wrong saying "on Flow", adding it on `FlowOps` will make it possible to `Flow[String].monitor` as well as `source.monitor`